mlir/test/lit.site.cfg.py.in

```diff
@@ -63,11 +63,43 @@ except KeyError:
     key, = e.args
     lit_config.fatal("unable to find %r parameter, use '--param=%s=VALUE'" % (key,key))
 
+# Helper function to decode arch to its features
+# Keep this in sync with mlir/lib/Dialect/Rock/Generator/AmdArchDb.cpp:mlir::rock::lookupArchInfo
+def set_arch_features(arch: str):
```

Comment:
You'll want comments on `AmdArchDb.cpp` as well, I think.

Reply:
I already have those!